hello-world/bundle.json

```diff
@@ -0,0 +1,4 @@
+{
+    "edgeworker-version": "0.0",
```

Comment:
Version 0.0 seems funny :)  Could this be 0.1 to be consistent with the other examples?